doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -112,6 +112,9 @@ nRF Desktop
 
   * Documentation for selective HID report subscription in :ref:`nrf_desktop_usb_state` using :kconfig:option:`CONFIG_DESKTOP_USB_SELECTIVE_REPORT_SUBSCRIPTION` option.
 
+* Removed configurations without a bootloader.
+  The B0 bootloader is enabled by default on all boards if the configuration fits in memory.
```

Comment:
```suggestion
  The B0 bootloader is enabled by default on all boards if the configuration with two image slots fits in memory.
  Alternatively, MCUboot bootloader with a single image slot and serial recovery is enabled.
  In case the configuration with the MCUboot does not fit in memory, no bootloader is enabled.
```